### PR TITLE
fa-1010 clean pom docs.developer

### PR DESCRIPTION
**Pull request by ICraig-CH**
__Short description outlining key changes/additions__

-cleaned pom
-updated parent pom to more recent version

__JIRA Ticket Number__

https://companieshouse.atlassian.net/browse/FA-1010

### Type of change

* [ ] Bug fix
* [ ] New feature
* [x] Breaking change

### Pull request checklist

* [x] I have added unit tests for new code that I have added
* [x] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -148,86 +116,96 @@
...
-    <spring-test.version>5.1.3.RELEASE</spring-test.version>
-    <spring-boot-maven-plugin.version>2.2.0.RELEASE</spring-boot-maven-plugin.version>
-    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-    <jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
-
-    <home.url></home.url>
-    <junit-platform-surefire-provider.version>1.2.0</junit-platform-surefire-provider.version>
-  </properties>
 
+  <build>
```
Comment (CH-JonathanBlake) [inline pom.xml:151]: Moved and indentation changes for some of the top lines (Unless I did a copy paste error)

File: pom.xml [hunk 2]
```diff
@@ -148,86 +116,96 @@
...
-  <packaging>jar</packaging>
-
-  <parent>
-    <artifactId>companies-house-parent</artifactId>
-    <groupId>uk.gov.companieshouse</groupId>
-    <version>1.2.0</version>
-    <relativePath/>
-  </parent>
 
-  <properties>
```
Comment (CH-JonathanBlake) [inline pom.xml:203]: Moved and blank lines removed.

File: pom.xml [hunk 3]
```diff
@@ -3,74 +3,46 @@
...
+    <structured-logging.version>1.5.0-rc2</structured-logging.version>
+    <thymeleaf-extras-java8time.version>3.0.1.RELEASE</thymeleaf-extras-java8time.version>
+    <thymeleaf-layout-dialect.version>2.3.0</thymeleaf-layout-dialect.version>
+    <web-security-java.version>1.3.8</web-security-java.version>
+    <oauth-signin-java.version>0.2.2</oauth-signin-java.version>
+    <spring-boot-dependencies.version>2.2.0.RELEASE</spring-boot-dependencies.version>
+    <spring-test.version>5.1.3.RELEASE</spring-test.version>
+    <spring-boot-maven-plugin.version>2.2.0.RELEASE</spring-boot-maven-plugin.version>
+    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+    <home.url></home.url>
```
Comment (CH-JonathanBlake) [inline pom.xml:43]: This looks wrong. Can someone confirm?

File: pom.xml [hunk 4]
```diff
@@ -3,74 +3,46 @@
...
 
-            <createAccount.url>${home.url}/create-account</createAccount.url>
-            <developerGuidelines.url>${home.url}/developer-guidelines</developerGuidelines.url>
-            <gettingStarted.url>${home.url}/getting-started</gettingStarted.url>
-            <manageApplications.url>${home.url}/manage-applications</manageApplications.url>
-            <home.url>${home.url}</home.url>
-            <piwik.siteId>//</piwik.siteId>
-            <piwik.url>//</piwik.url>
+  <artifactId>docs.developer.ch.gov.uk</artifactId>
+  <version>unversioned</version>
```
Comment (CH-JonathanBlake) [inline pom.xml:10]: Should we be finally adding a version number?

File: pom.xml (hunk 3 above)
Comment (ICraig-CH) [reply pom.xml:43]: I left the contents of the pom alone and focused on the structure, this is how that entry has been for a while I think

File: pom.xml (hunk 4 above)
Comment (Zain-Abbas1) [reply pom.xml:10]: No I think it remains unversioned

File: pom.xml [hunk 5]
```diff
@@ -148,86 +116,96 @@
...
-  <name>docs.developer.ch.gov.uk</name>
-
-  <packaging>jar</packaging>
-
-  <parent>
-    <artifactId>companies-house-parent</artifactId>
-    <groupId>uk.gov.companieshouse</groupId>
-    <version>1.2.0</version>
-    <relativePath/>
-  </parent>
```
Comment (Zain-Abbas1) [inline pom.xml:149]: Blank lines

File: pom.xml [hunk 6]
```diff
@@ -3,74 +3,46 @@
...
-  </build>
+    <structured-logging.version>1.5.0-rc2</structured-logging.version>
+    <thymeleaf-extras-java8time.version>3.0.1.RELEASE</thymeleaf-extras-java8time.version>
+    <thymeleaf-layout-dialect.version>2.3.0</thymeleaf-layout-dialect.version>
+    <web-security-java.version>1.3.8</web-security-java.version>
+    <oauth-signin-java.version>0.2.2</oauth-signin-java.version>
+    <spring-boot-dependencies.version>2.2.0.RELEASE</spring-boot-dependencies.version>
+    <spring-test.version>5.1.3.RELEASE</spring-test.version>
+    <spring-boot-maven-plugin.version>2.2.0.RELEASE</spring-boot-maven-plugin.version>
+    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
```
Comment (Zain-Abbas1) [inline pom.xml:42]: Maybe worth collecting the maven properties together & same with the Spring ones under `<!-- Spring -->`

File: pom.xml [hunk 7]
```diff
@@ -3,74 +3,46 @@
...
-            <manageApplications.url>${home.url}/manage-applications</manageApplications.url>
-            <home.url>${home.url}</home.url>
-            <piwik.siteId>//</piwik.siteId>
-            <piwik.url>//</piwik.url>
+  <artifactId>docs.developer.ch.gov.uk</artifactId>
+  <version>unversioned</version>
+  <packaging>jar</packaging>
+  <name>docs.developer.ch.gov.uk</name>
+  <description>Static content for the developer hub,
+    such as "getting started" and "introduction" documentation
```
Comment (Zain-Abbas1) [inline pom.xml:14]: Description can change to `Static general documentation for....`

File: pom.xml [hunk 8]
```diff
@@ -3,74 +3,46 @@
...
-        </dependencies>
-        <groupId>org.apache.maven.plugins</groupId>
-        <version>${maven-surefire-plugin.version}</version>
-      </plugin>
+  <parent>
+    <artifactId>companies-house-parent</artifactId>
+    <groupId>uk.gov.companieshouse</groupId>
+    <version>1.3.0</version>
+    <relativePath/>
+  </parent>
```
Comment (Zain-Abbas1) [inline pom.xml:22]: `<parent>` to be moved above just below `<modelVersion>` please

File: pom.xml (hunk 5 above)
Comment (Zain-Abbas1) [inline pom.xml:149]: Blank lines

File: pom.xml (hunk 6 above)
Comment (Zain-Abbas1) [inline pom.xml:42]: Maybe worth collecting similar properties for Spring under `<!-- Maven -->` & Spring as well.

File: pom.xml (hunk 7 above)
Comment (Zain-Abbas1) [inline pom.xml:14]: Descrition can be changed to `Static general documentation content for...` deleting the `such as "getting started" and "introduction" documentation`.

**Review by Zain-Abbas1 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 8 above)
Comment (ICraig-CH) [reply pom.xml:22]: I saw that on the guide but on the repositories that Steve Roberts worked on the parent was were I put it, I checked the ones he worked on as someone suggested yesterday should I still change it?

Comment (Zain-Abbas1) [reply pom.xml:22]: Yes please, the guide suggests that is the preferred format

Comment (ICraig-CH) [reply pom.xml:22]: okay :) I will change applications.api as well

File: pom.xml (hunk 6 above)
Comment (ICraig-CH) [reply pom.xml:42]: Will do that now, I will add one for Java as well

File: pom.xml [hunk 9]
```diff
@@ -3,74 +3,50 @@
...
-          <dependency>
-            <artifactId>junit-platform-surefire-provider</artifactId>
-            <groupId>org.junit.platform</groupId>
-            <version>${junit-platform-surefire-provider.version}</version>
-          </dependency>
-        </dependencies>
-        <groupId>org.apache.maven.plugins</groupId>
-        <version>${maven-surefire-plugin.version}</version>
-      </plugin>
+  <name>docs.developer.ch.gov.uk</name>
```
Comment (Zain-Abbas1) [inline pom.xml:23]: Can we remove the blank lines between lines 16-24

**Review by Zain-Abbas1 — APPROVED** [review]

**Review by ch-rfishwick — APPROVED** [review]

**Pull request merged**